modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -65,115 +65,59 @@ endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 . Set the `$RELEASE_IMAGE` variable by running the following command:
 +
-[source,terminal] 
+[source,terminal]
 ----
 $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
 endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
+//These conditions could be applied to only what is different (`--cloud=<blah>`) to avoid maintenance difficulties.
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image by running the following command:
 +
 [source,terminal]
 ifdef::aws-sts[]
 ----
 $ oc adm release extract \
---credentials-requests \
---cloud=aws \
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
---from=quay.io/<path_to>/ocp-release:<version>
+  --credentials-requests \
+  --cloud=aws \
+  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
```

Comment:
Pre-existing content, but `<path_to_directory_with_list_of_credentials_requests>/credrequests` feels like it's over-nested.  It's often worth extracting to a clean directory for easy post-cred-creation cleanup, but even then, using a directory is optional.  And when folks do set `--to`, they'll just want to pick some doesn't-exist-yet path, it doesn't need to end with `/credrequests`.  Maybe we can just use `<path_to_directory_with_list_of_credentials_requests>`?

Reply:
As written, this would make `credrequests/credrequests`, wouldn't it?